nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
 
+  // Reset profiling timers and counters within event_delivery_manager
+  kernel().event_delivery_manager.reset_timers_counters();
```

Comment:
I would move it, because now - if you think of using sionlib as I/O backend - you have to call `run` instead of `simulate` to record all data in one container. And if you want to have separate measurements for each invocation of `run` (what would have been `simulate` before) you need the reset within `run`.